clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -3003,6 +3003,12 @@ static void RenderFloatingPointOptions(const ToolChain &TC, const Driver &D,
       RoundingMathPresent = false;
       break;
 
+    case options::OPT_fcuda_flush_denormals_to_zero:
+    case options::OPT_fgpu_flush_denormals_to_zero:
+      DenormalFPMath = llvm::DenormalMode::getPositiveZero();
+      DenormalFP32Math = llvm::DenormalMode::getPositiveZero();
+      break;
```

Comment:
Could you add a driver test verifying the expect values when the options are set?